libs/api/shared/src/lib/kernel/graphql/subscriptions/observable-to-asynciterable.helper.ts

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
```

Comment:
Please make `eslint-disable` line specific as allowing no-non-null-assertions om the whole file removes the type-checking for accidentially created ones too.